quickwit/quickwit-indexing/src/source/kafka_source.rs

```diff
@@ -49,7 +49,10 @@ use tracing::{debug, info, warn};
 
 use crate::actors::DocProcessor;
 use crate::models::{NewPublishLock, PublishLock};
-use crate::source::{BatchBuilder, Source, SourceContext, SourceRuntimeArgs, TypedSourceFactory};
+use crate::source::{
+    BatchBuilder, Source, SourceContext, SourceRuntimeArgs, TypedSourceFactory,
+    EMIT_BATCHES_TIMEOUT,
+};
 
 /// Number of bytes after which we cut a new batch.
 ///
```

Comment:
I've noticed that here and in other sources we are using something like `BATCH_NUM_BYTES_LIMIT: u64 = 5_000_000`.
1. I'm happy to replace it with Bytsize
2. should it be `5Mb` (as it here) or `5Mib`(like in [mod.rs](https://github.com/quickwit-oss/quickwit/pull/4112/files#diff-a2df65a154da2a6ab79da9866d7a5dfca567d9951ee3bd3eb2687fe3d4681801))?

Reply:
1. Yes, good idea. Let's use a shared `Bytsize ` constant defined in the source module for all sources, as you suggested.

2. Up to you :)